nuxt/components/form/Form.vue

```diff
@@ -63,19 +74,57 @@ const props = withDefaults(defineProps<Props>(), {
   submitName: undefined,
 })
 
+// data
+const form = reactive({
+  turnstileKey: '',
+})
+
+// vuelidate
+const rules = {
+  turnstileKey: {
+    required,
+  },
+}
+const v$ = useVuelidate(rules, form)
+
+// methods
 const emit = defineEmits<{
   (e: 'click'): void
   (e: 'submit', event: Event): void
 }>()
 
+const store = useMaevsiStore()
 const { t } = useI18n()
 
+// methods
+const onSubmit = (e: Event) => {
+  storeTurnstileKey()
+  emit('submit', e)
+}
```

Comment:
```suggestion
const emit = defineEmits<{
  (e: 'click'): void
  (e: 'submit', event: Event): void
}>()

const store = useMaevsiStore()
const { t } = useI18n()

// data
const form = reactive({
  turnstileKey: '',
})

// vuelidate
const rules = {
  turnstileKey: {
    required,
  },
}
const v$ = useVuelidate(rules, form)

// methods
const onSubmit = (e: Event) => {
  storeTurnstileKey()
  emit('submit', e)
}
```

corrected [order of elements](https://github.com/maevsi/maevsi/blob/master/CONTRIBUTING.md#script-setup-order)
1. emit
2. uses
3. data
4. vuelidate
5. methods (deduplicated comment)